stable/insights-agent/templates/helm-hook/job.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  {{- include "metadata" "cronjob-executor" }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+spec:
+  ttlSecondsAfterFinished: 30
+  backoffLimit: {{ .Values.cronjobs.backoffLimit }}
+  template:
+    spec:
+      restartPolicy: Never
+      serviceAccountName: {{ include "insights-agent.fullname" . }}-cronjob-executor
+      containers:
+      - name: start-job-from-cronjob
+        image: bitnami/kubectl:{{ .Capabilities.KubeVersion.Major }}.{{ .Capabilities.KubeVersion.Minor }}
+        command: ["bash"]
+        args:
+          - -c
+          - |
+            kubectl get cj --selector=app=insights-agent -o wide | grep insights | cut -d' ' -f1 | xargs -I~job~ sh -c "kubectl delete job ~job~ || echo 'Does Not Exist'"
```

Comment:
IIUC, this would delete the the cronjob job history, which the user might want to keep around

Maybe we could give the job a name like `$reportname-init`, and grep for that? Or add a label so we could select specifically for the init jobs before deleting them?

Reply:
This will only delete the jobs that's names EXACTLY match the cronjob, so just the ones that this is creating not the ones that the CronJobs create.

Reply:
Because when the CronJob creates a job it appends the time at the end of the Job name.

Reply:
👍 thanks for the explanation!